internal/common/metrics.go

```diff
@@ -63,12 +63,10 @@ func sendStatsdMetrics(metricPerMinute int, duration time.Duration) error {
 		select {
 		case <-ticker.C:
 			for time := 0; time < metricPerMinute; time++ {
-				go func(time int) {
-					client.Inc(fmt.Sprintf("%v", time), int64(time), 1.0)
-				}(time)
+				client.Inc(fmt.Sprintf("%v", time), int64(time), 1.0)
```

Comment:
This might involve in [Adam PR a little bit](https://github.com/aws/amazon-cloudwatch-agent-test/pull/152#discussion_r1122264534). The `client.Inc` should be quickly when sending over UDP and does not need a go-routine and can be quickly handle in 1 minute interval even with 50,000 metrics

Reply:
The downside is minimum so it can be safely ignored since in terms of more than 50,000 metrics, even if 20,000 being send, it should be able to stress the CWA to its maximum capacity based on the limited buffer (10000). (So dropping 30,000 in sending the next minute is fine too)